extractor/src/test/java/org/schabi/newpipe/extractor/services/youtube/YoutubePlaylistExtractorTest.java

```diff
@@ -365,18 +366,18 @@ public void testUploaderUrl() throws Exception {
         @Test
         public void testUploaderName() throws Exception {
             final String uploaderName = extractor.getUploaderName();
-            assertTrue(uploaderName, uploaderName.contains("CrashCourse"));
+            ExtractorAsserts.assertContains("CrashCourse", uploaderName);
         }
 
         @Test
         public void testUploaderAvatarUrl() throws Exception {
             final String uploaderAvatarUrl = extractor.getUploaderAvatarUrl();
-            assertTrue(uploaderAvatarUrl, uploaderAvatarUrl.contains("yt"));
+            ExtractorAsserts.assertContains("yt", uploaderAvatarUrl);
         }
 
         @Test
         public void testStreamCount() throws Exception {
-            assertTrue("Error in the streams count", extractor.getStreamCount() > 40);
+            assertTrue(extractor.getStreamCount() > 40, "Error in the streams count");
```

Comment:
Can't you use `ExtractorAsserts.assertAtLeast` here? Maybe also in other places.